### ISSUE TITLE
3.37.0 Broke annotations on category x-axis 

### ISSUE THREAD

**Issue by brodrigz** [apexcharts/apexcharts.js#3654]
### Description
Annotations on an category x-axis are no longer rendered on charts in the latest 3.37.0 release.

This can be observed in the [documentation ](https://apexcharts.com/javascript-chart-demos/column-charts/column-with-rotated-xaxis-labels/)

### Steps to Reproduce

1. Create a chart with the following code on version 3.37.0 :
```
var options = {
          series: [{
          name: 'Servings',
          data: [44, 55, 41, 67, 22, 43, 21, 33, 45, 31, 87, 65, 35]
        }],
          annotations: {
          points: [{
            x: 'Bananas',
            seriesIndex: 0,
            label: {
              borderColor: '#775DD0',
              offsetY: 0,
              style: {
                color: '#fff',
                background: '#775DD0',
              },
              text: 'Bananas are good',
            }
          }]
        },
        chart: {
          height: 350,
          type: 'bar',
        },
        plotOptions: {
          bar: {
            borderRadius: 10,
            columnWidth: '50%',
          }
        },
        dataLabels: {
          enabled: false
        },
        stroke: {
          width: 2
        },
        
        grid: {
          row: {
            colors: ['#fff', '#f2f2f2']
          }
        },
        xaxis: {
          labels: {
            rotate: -45
          },
          categories: ['Apples', 'Oranges', 'Strawberries', 'Pineapples', 'Mangoes', 'Bananas',
            'Blackberries', 'Pears', 'Watermelons', 'Cherries', 'Pomegranates', 'Tangerines', 'Papayas'
          ],
          tickPlacement: 'on'
        },
        yaxis: {
          title: {
            text: 'Servings',
          },
        },
        fill: {
          type: 'gradient',
          gradient: {
            shade: 'light',
            type: "horizontal",
            shadeIntensity: 0.25,
            gradientToColors: undefined,
            inverseColors: true,
            opacityFrom: 0.85,
            opacityTo: 0.85,
            stops: [50, 0, 100]
          },
        }
        };

        var chart = new ApexCharts(document.querySelector("#chart"), options);
        chart.render();
```
      

### Expected Behavior
Annotations are rendered on the chart
### Actual Behavior
Annotations are NOT rendered on the chart

#### PREVIOUS VERSION:
![image](https://user-images.githubusercontent.com/57366583/218559419-2c277c62-4252-4471-99cd-908dae4a0348.png)

#### 3.37.0:
![image](https://user-images.githubusercontent.com/57366583/218559514-9b01b2d5-7c7c-42c0-a942-28467398b0f0.png)

### Reproduction Link
https://apexcharts.com/javascript-chart-demos/column-charts/column-with-rotated-xaxis-labels/

**Comment by canblt** [comment]
any updates for that issue ?

**Comment by mygpse** [comment]
Move the code snippet version 3.37.*
if (w.config.annotations.position === 'back') {  w.globals.dom.Paper.add(w.globals.dom.elAnnotations);  me.annotations.drawAxesAnnotations();  }
from line 32183 to line 32137.

position: 'back' no work
![image](https://user-images.githubusercontent.com/3582197/224909658-b171ee4a-d244-47ed-9963-8e85ebac71d9.png)

position: 'back' will work
![image](https://user-images.githubusercontent.com/3582197/224909354-71c23282-6462-42ce-8245-91632b3b2b11.png)

**Comment by pav-stastny** [comment]
> Move the code snippet (this works on 3.37.0) if (w.config.annotations.position === 'back') { w.globals.dom.Paper.add(w.globals.dom.elAnnotations); me.annotations.drawAxesAnnotations(); } from line 32183 to line 32137.


This is not a proper fix. What if we are using normal npm module? Then what? IMO this is uninentional breaking change.

**Comment by junedchhipa** [comment]
The original issue has been fixed (as tested on v4.0.0).
Please open a new issue if you still see the problem.

**Comment by Guziq** [comment]
It still doesn't work on 4.0.0 (works on `3.36.0`), but I will create a new ticket as suggested.
